django_nextjs/render.py

```diff
@@ -49,7 +49,9 @@ def _get_nextjs_request_cookies(request: HttpRequest):
     https://docs.djangoproject.com/en/3.2/ref/csrf/#is-posting-an-arbitrary-csrf-token-pair-cookie-and-post-data-a-vulnerability
     """
     unreserved_cookies = {k: v for k, v in request.COOKIES.items() if k and not morsel.isReservedKey(k)}
-    return {**unreserved_cookies, settings.CSRF_COOKIE_NAME: get_csrf_token(request)}
+    if all([ENSURE_CSRF_TOKEN, settings.CSRF_COOKIE_NAME not in unreserved_cookies]):
```

Comment:
This is technically correct, but I prefer using a simple `and` between the two conditions for better readability.